src/pages/dashboard/index.svelte

```diff
@@ -72,15 +72,22 @@
         [currentAccount, stateREX],
         ([$currentAccount, $stateREX]) => {
             if ($currentAccount && $currentAccount.rex_info && $stateREX && $stateREX.value) {
-                return $stateREX.value * $currentAccount.rex_info.rex_balance.value
+                if ($stateREX.value === 0.0001) {
```

Comment:
Out of curiosity, what is going on here? Why do we check for a 0.0001 value?

Reply:
`0.0001` is the floor of the asset, and on Telos, the value is much lower than that. It's an issue with the library we are using, which I need to fix - but didn't want that to get in the way of releasing these balance updates.

Reply:
Ah ok makes sense!